Add next-gen future documentation (#761)
slackapi/bolt-python: docs/_future/manifest.md

---
title: Manifest
lang: en
slug: manifest
order: 2
layout: future
---

<div class="section-content">

> Your project should contain a `manifest.json` file that defines your app's manifest, which is where you can configure your application name and scopes, declare the functions your app will use, and more.

Reviewer: ```suggestion
Your project should contain a `manifest.json` file that defines your app's manifest. This is where you'll configure your application name and scopes, declare the functions your app will use, and more.
```

> Refer to the <a href="https://api.slack.com/reference/manifests" target="_blank">App Manifest documentation</a> to learn

Reviewer 2: Should we link to https://api.slack.com/reference/manifests, or maybe better to the next-gen page on manifest properties: https://api.slack.com/future/manifest#manifest-properties ?

Author: I'll add both 💯

about the available manifest configurations.

Notably, the App Manifest informs Slack of the definitions for:

* [Functions](/bolt-python/future/concepts#manifest-functions)
* [Workflows](/bolt-python/future/concepts#manifest-workflows)

> `manifest.json` is located at the top level of <a href="https://github.com/slack-samples/bolt-python-starter-template" target="_blank">our example projects</a>, along with a `triggers` folder containing

Reviewer 2: Does the link to the template need to also link to the `future` branch?

Author: Very good catch 💯
